lang/src/org/partiql/lang/util/ExprValueFormatter.kt

```diff
@@ -41,25 +38,26 @@ class ConfigurableExprValueFormatter(private val config: Configuration) : ExprVa
         fun recursivePrettyPrint(value: ExprValue) {
             when (value.type) {
 
-                MISSING                                    -> out.append(MISSING_STRING)
-                NULL                                       -> out.append(NULL_STRING)
+                ExprValueType.MISSING                                    -> out.append(MISSING_STRING)
+                ExprValueType.NULL                                       -> out.append(NULL_STRING)
 
-                BOOL                                       -> out.append(value.scalar.booleanValue().toString())
+                ExprValueType.BOOL                                       -> out.append(value.scalar.booleanValue().toString())
 
-                INT, DECIMAL                               -> out.append(value.scalar.numberValue().toString())
+                ExprValueType.INT, ExprValueType.DECIMAL                               -> out.append(value.scalar.numberValue().toString())
 
-                STRING                                     -> out.append("'${value.scalar.stringValue()}'")
+                ExprValueType.STRING                                     -> out.append("'${value.scalar.stringValue()}'")
 
-                DATE                                       -> out.append(value.scalar.dateValue().toString())
+                ExprValueType.DATE                                       -> out.append(value.scalar.dateValue().toString())
 
-                TIME                                       -> out.append(value.scalar.timeValue().toString())
+                ExprValueType.TIME                                       -> out.append(value.scalar.timeValue().toString())
 
                 // fallback to an Ion literal for all types that don't have a native PartiQL representation
-                FLOAT, TIMESTAMP, SYMBOL, CLOB, BLOB, SEXP -> prettyPrintIonLiteral(value)
+                ExprValueType.FLOAT, ExprValueType.TIMESTAMP, ExprValueType.SYMBOL,
+                ExprValueType.CLOB,ExprValueType. BLOB, ExprValueType.SEXP -> prettyPrintIonLiteral(value)
 
-                LIST                                       -> prettyPrintContainer(value, "[", "]")
-                BAG                                        -> prettyPrintContainer(value, "<<", ">>")
-                STRUCT                                     -> prettyPrintContainer(value, "{", "}") { v ->
+                ExprValueType.LIST                                       -> prettyPrintContainer(value, "[", "]")
+                ExprValueType.BAG                                        -> prettyPrintContainer(value, "<<", ">>")
+                ExprValueType.STRUCT                                     -> prettyPrintContainer(value, "{", "}") { v ->
```

Comment:
nit: in the past we had aligned the `when` expression branch arrows together, which proved tedious (e.g. if a new clause gets added that's longer, have to adjust every branch). Consider aligning to after the the LHS expression.

Reply:
Fixed.